src/components/ConversionFormOverlay/style.css

```diff
@@ -0,0 +1,56 @@
+.container{
+    height: 100vh;
+    width: 100%;
+    background-color: rgba(231, 228, 242, 1);
+    padding-left: 30px;  
+    padding-top: 30px;
+}
+
+.container .loading-icon, .container.loading-text {
+    text-align: center;
+}
+
+.upload {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+
+.upload-container {
+    display: flex;
+    flex-direction: row;
+}
+
+.selector-box {
+    width: 200px;
+    margin-right: 20px;
+}
```

Comment:
this class doesn't seem to be used?

Reply:
it sizes the Select element for choosing simulation engine